docker/rootfs/etc/s6-overlay/s6-rc.d/prepare/50-ipv6.sh

```diff
@@ -31,9 +32,17 @@ process_folder () {
 		echo "$(sed -E "$SED_REGEX" "$FILE")" > $FILE
 	done
 
-	# ensure the files are still owned by the npm user
-	chown -R "$PUID:$PGID" "$1"
+
+		# ensure the files are still owned by the npm user
+		chown -R "$PUID:$PGID" "$1"
```

Comment:
Only this line in this file should be wrapped in the IF statement. You still want the SED above to function.